backend/router/paymentRouter.js

```diff
@@ -3,13 +3,32 @@ const {
   createSubscription,
   getRazorpayApiKey,
   verifySubscription,
+  createPlan,
+  deletePlan,
+  updatePlan,
+  getPlans,
 } = require("../controller/paymentController");
 const jwtAuth = require("../middleware/jwtAuth.js");
+const authAdmin = require("../middleware/authAdmin");
```

Comment:
After rewriting the auth middleware fix this as well

Reply:
replace with authorizeRoles